extensions-contrib/kafka-emitter/pom.xml

```diff
@@ -81,7 +81,12 @@
       <artifactId>jackson-core</artifactId>
       <scope>provided</scope>
     </dependency>
-
+    <dependency>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <version>2.10.5</version>
```

Comment:
can we inherit this version from the common pom?

Reply:
Updated it @xvrl